bin/server.dart

```diff
@@ -16,6 +16,7 @@ Future<void> main() async {
     o.release = Platform.environment['VERSION'];
```

Comment:
These are now read by Sentry itself, though with a `SENTRY_` prefix.

Reply:
Nice. I'd need to change how I build the docker image too so I'll leave it as-is for now